src/main/java/com/ita/if103java/ims/handler/GlobalExceptionHandler.java

```diff
@@ -48,7 +50,9 @@
     @ExceptionHandler({ImpossibleWarehouseAdviceException.class,
         MaxWarehouseDepthLimitReachedException.class,
         MaxWarehousesLimitReachedException.class,
-        AssociateLimitReachedException.class})
+        AssociateLimitReachedException.class,
+        UserLimitReachedException.class,
+        UpgradationException.class})
```

Comment:
If this and below fix related to https://trello.com/c/Ooc52I4E

Then it is not what I expected. But let it be